simpegEM/FDEM/FDEM.py

```diff
@@ -8,6 +8,102 @@ def omega(freq):
     """Change frequency to angular frequency, omega"""
     return 2.*np.pi*freq
 
+def getSource(self,freq):
```

Comment:
It isn't the RHS. The RHS depends on which implementation you are using, and somehow modifies the source. Ie.  Problem_e.getRHS = -1j*omega(freq)*j_s